File: isPalindrome.js
var isPalindrome = function(x) {
    x = String(x);
    var reverse = ''
    for (var i=x.length-1; i>=0; i--){
        reverse += x[i];
    }
    console.log(reverse)
    if (reverse === x){
        return true;
    }
    else{
        return false
    }
    
};

File: is_palindrome.py
def isPalindrome(x):
    """
    :type x: int
    :rtype: bool
    """
    reverse = ''
    x=str(x)
    for i in range (len(x)-1,-1,-1):
        reverse+=x[i]
    print (reverse)
    return reverse == x
